### DISCUSSION TITLE
buddhism/level_1/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#48] [General]
# buddhism/level_1/

https://shanenull.com/buddhism/level_1/

**Comment by shane0** [comment]
Add page numbers for awareness types
Find notes on tips for troubleshooting breathing
Use watch to track heart rate lowering

**Comment by shane0** [comment]
# week 9 

nov 7 topic b. vegetarian

attempting to practice auspicious days gradually, full fast or fast after 12 or fast from eating animals

checkout food books from my notes

vegetarian athletes https://gamechangersmovie.com/
